packages/tools/client-debugger/client-debugger/src/DevtoolsLogger.ts

```diff
@@ -76,7 +76,7 @@ export class FluidDebuggerLogger extends TelemetryLogger {
 	};
 
 	/**
-	 * Posts a list of {@link IDevtoolsMessage} to the window (globalThis). It will be send
+	 * Posts a list of {@link TelemetryHistory.Message} to the window (globalThis). It will be send
```

Comment:
```suggestion
	 * Posts a {@link TelemetryHistory.Message} to the window (globalThis), containing the complete history of telemetry events. It will be send
```